kuryr_kubernetes/cni/daemon/service.py

```diff
@@ -70,6 +72,17 @@ def _prepare_request(self):
                   params.CNI_COMMAND, params)
         return params
 
+    def _update_metrics(self, command, error, duration):
+        """Add a new metric value to the shared metrics dict"""
+        params = self._prepare_request()
```

Comment:
Don't we have here possibility for exception with reading CNI params, similar how it is done in add/delete methods?

Reply:
yup, thanks.